docs/run-platform-server/monitoring.mdx

```diff
@@ -7,11 +7,54 @@ import { CodeExample } from "@site/src/components/CodeExample";
+
+- `go_`: golang specfici runtime performance
+- `horizon_txsub_`: attributes of horizon transaction submission sub system if enabled.
+- `horizon_stellar_core_`: runtime attributes of stellar network reported by the captive core or a remote stellar core which horizon is configured.
+- `horizon_order_book_`: runtime attributes of the in memory order book maintained by horizon of the current stellar network
+- `horizon_log_`: counters of how many log messages printed at each severity level
+- `horizon_ingest_`: performance measurements and stateful aspects of horizon's internal ingestion sub system
+- `horizon_http_`: statistics and measurements of horizon's HTTP API service, all aspects of request/response load and timings.
+- `horizon_history_`: statistics on horizon ingested historical ledgers
+- `horizon_db_`: measurements on database performance, query times per endpoints, pooling stats
+- `process_`: generic host machine compute measurements
+
+And for each key, there will be possibility of 0 or more labels, the serialized output(exposition) format follows this template:
+
+```
+<metrics_key_name>{label_1="value",label_2="value",,,} <value>
+```
+
+Rather than listing all individual metrics keys in docs, as they change often, the recommendation is to perform an HTTP GET against the horizon metrics endpoint,`localhost:<ADMIN_PORT>/metrics`, using any http client(browser, curl, wget, etc) and the response will have the metrics keys and additional meta information on each metric key for description and type(counter, guage, histogram, summary), as an example for one key, `horizon_http_requests_duration_seconds`:
+
+```
+# HELP horizon_http_requests_duration_seconds HTTP requests durations, sliding window = 10m
+# TYPE horizon_http_requests_duration_seconds summary
+horizon_http_requests_duration_seconds{method="GET",route="/",status="200",streaming="false",quantile="0.5"} 0.000186958
+horizon_http_requests_duration_seconds{method="GET",route="/",status="200",streaming="false",quantile="0.9"} 0.00043625
+horizon_http_requests_duration_seconds{method="GET",route="/",status="200",streaming="false",quantile="0.99"} 0.000645
+...
+
+```
 
 ### Queries
 
-Build queries that highlight performance of a given Horizon deployment. Refer to Stellar's [Grafana Horizon Dashboard](https://grafana.com/grafana/dashboards/13793-stellar-horizon/) for examples of metrics queries to derive application performance:
+Build queries against the metrics data model to highlight performance of a given Horizon deployment. Refer to Stellar's [Grafana Horizon Dashboard](https://grafana.com/grafana/dashboards/13793-stellar-horizon/) for examples of metrics queries to derive application performance:
```

Comment:
I can't seem to find some of [these](https://github.com/stellar/stellar-docs/pull/198/files#diff-eaaac1bf290b3eed0ad14d5e03bac252f1474e499b377ae807e87ecb612969f3R64-R67) on the Grafana dashboard. Are these queries provided as examples, or can I find the actual queries somewhere?

Reply:
What I'm aware of was dowloading the dashboard .json from https://grafana.com/grafana/dashboards/13793-stellar-horizon/, then can copy the metrics queries defined on panels with titles that should align fairly closes to the example queries listed here, if there is a gap where queries in there don't overlap enough with the example queries listed here, there's no other published grafana dashboards, so, would probably just remove from examples, which ones did you notice?

Reply:
I couldn't find these
- Number of rate-limited requests.
- List of rate-limited IPs.
- Unique IPs.
- The most popular SDKs/apps sending requests to a given Horizon node.
- Average ingestion time of a transaction